<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Доступы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QLineEdit" name="le_worker">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cb_position">
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtBottom</enum>
     </property>
     <property name="minimumContentsLength">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QTableWidget" name="tw_access">
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>18</number>
     </attribute>
     <column>
      <property name="text">
       <string>Класс</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Атрибут 1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Атрибут 2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Значение</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_4">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_2">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Выберитк работника или групу</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_view_worker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>-109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_position</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>ui_select_position()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_add_position()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>-103</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tw_access</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>Dialog</receiver>
   <slot>ui_double_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>-304</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>ui_del_position()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>-121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ui_view_worker()</slot>
  <slot>ui_select_position()</slot>
  <slot>ui_add_position()</slot>
  <slot>ui_double_click()</slot>
  <slot>ui_del_position()</slot>
 </slots>
</ui>
